homework/src/index.js

```diff
@@ -30,18 +33,70 @@
     return elem;
   }
 
+  function renderRepository(container, repository) {
+    container.innerHTML = '';
+    //const root = document.getElementById('root');
+    const table = createAndAppend('table', container);
+    const tBody = createAndAppend('tBody', table);
+    const trRepository = createAndAppend('tr', tBody);
+    trRepository.className = "table-info";
+    createAndAppend('td', trRepository, {html: "Repository:"});
+    //createAndAppend('td', trRepository, {html: repository.name});
+    const link = createAndAppend('a', trRepository,{html: repository.name});
```

Comment:
A `tr` element should only have `td` or `th` elements as its children. It should not contain anything else like the `a` tag here.

```html
<table>
  <tbody>
    <tr class="table-info">
      <td>Repository:</td>
      <a href="https://github.com/HackYourFuture/AngularJS">AngularJS</a>  <-- invalid HTML
    </tr>
    <tr class="table-info">
      <td>Description:</td>
      <td>Class material from angularJS module</td>
    </tr>
    <tr class="table-info">
      <td>Forks:</td>
      <td>2</td>
    </tr>
    <tr class="table-info">
      <td>Updated:</td>
      <td>2018-04-06T17:18:06Z</td>
    </tr>
  </tbody>
</table>
```